constants.js

```diff
@@ -1,3 +1,10 @@
 var xdg = require('xdg-basedir');
-exports.LOCAL_FILE_PATH = xdg.data + '/cube/times.csv';
-exports.PUSHED_FILE_PATH = xdg.data + '/cube/pushed.csv';
+var path = require('path');
+
+var localSaveDir = path.join(xdg.data, '/cube');
+var localFilePath = path.join(localSaveDir, 'times.csv');
+var pushedFilePath = path.join(localSaveDir, 'pushed.csv');
+
+exports.LOCAL_SAVE_DIR = localSaveDir;
+exports.LOCAL_FILE_PATH = localFilePath;
+exports.PUSHED_FILE_PATH = pushedFilePath;
```

Comment:
Although these changes are fine, I don't see the need for them. Can you please explain your rationale? Since the path is basically a constant for a particular OS, I don't see any reason to introduce the three new function calls. Thanks! :slightly_smiling_face:

Reply:
Your `LOCAL_SAVE_DIR` idea is required! Thanks for that, but I would just define it as a constant instead of using `path.join`.

Reply:
Hi,

This is a best practice in my opinion since it ensures that the path is not malformed or at least normalized. Using `path` should handle all use cases but on the other hand it boils down to preference.

I will honor your opinion on this and change it accordingly.

Reply:
Hmm, I think `xdg-basedir` is dependable. :slightly_smiling_face: I know what they say about micro-optimizations, but it's still a few function calls I would avoid.

(If things go bad and a bug comes up in the future, because of this, then I will be sure to own up to my error in judgment :smile_cat:  )